client/uploadScrambles.js

```diff
@@ -105,12 +105,11 @@ function extractJsonFromZip(filename, zipId, pw, cb) {
         } else {
           cb("Wrong password");
         }
-        return;
-      } else {
-        throw error;
       }
+      console.error("Meteor.call() error: " + err);
```

Comment:
I would like to log all errors. I don't think an incorrect password counts here, though, as we are notifying the user anyways.